ouroboros-network/ouroboros-network.cabal

```
@@ -55,6 +55,20 @@ library
   -- This has to be tidied up once the design becomes clear.
   exposed-modules:
     Control.Concurrent.Class.MonadSTM.Strict.TMergeVar
+    Ouroboros.Cardano.Diffusion.Configuration
+    Ouroboros.Cardano.Diffusion.Handlers
+    Ouroboros.Cardano.Diffusion.Policies
+    Ouroboros.Cardano.Network.ArgumentsExtra
+    Ouroboros.Cardano.Network.LedgerPeerConsensusInterface
+    Ouroboros.Cardano.Network.PeerSelection.Governor.Monitor
+    Ouroboros.Cardano.Network.PeerSelection.Governor.PeerSelectionActions
+    Ouroboros.Cardano.Network.PeerSelection.Governor.PeerSelectionState
+    Ouroboros.Cardano.Network.PeerSelection.Governor.Types
+    Ouroboros.Cardano.Network.PeerSelection.PeerChurnArgs
+    Ouroboros.Cardano.Network.PublicRootPeers
+    Ouroboros.Cardano.Network.Types
+    Ouroboros.Cardano.PeerSelection.Churn
+    Ouroboros.Cardano.PeerSelection.PeerSelectionActions
```

This will be simpler (it's so good that `hls` can assist in something like that :)).

```suggestion
    Cardano.Diffusion.Configuration
    Cardano.Diffusion.Handlers
    Cardano.Diffusion.Policies
    Cardano.Network.ArgumentsExtra
    Cardano.Network.LedgerPeerConsensusInterface
    Cardano.Network.PeerSelection.Governor.Monitor
    Cardano.Network.PeerSelection.Governor.PeerSelectionActions
    Cardano.Network.PeerSelection.Governor.PeerSelectionState
    Cardano.Network.PeerSelection.Governor.Types
    Cardano.Network.PeerSelection.PeerChurnArgs
    Cardano.Network.PublicRootPeers
    Cardano.Network.Types
    Cardano.PeerSelection.Churn
    Cardano.PeerSelection.PeerSelectionActions
```